src/opinions/createOpinion.ts

```diff
@@ -0,0 +1,29 @@
+import { Response, Request } from "express";
+import { StatusCodes } from "http-status-codes";
+import mongoose from "mongoose";
+
+import opinionModel from "../../models/opinion.model";
+import validateOpinion from "./validateOpinion";
+
+const createOpinion = async (req: Request, res: Response) => {
+  const { error } = validateOpinion(req.body);
+  if (error) return res.status(400).send(error.details[0].message);

```

Comment:
Zmień, proszę status używając StatusCodes, żeby było to spójne w całym projekcie :)